### PR TITLE
web: Migrate to npm workspaces

### PR DESCRIPTION
**Pull request by relrelb**
`lerna` is a bit stale, and as such it currently has some outdated
dependencies which Dependabot warns on.
Fortunately, npm 7 supports monorepos natively, via "workspaces". So
simply replace `lerna` with this feature. The migration is pretty
neat and requires a very little invervention.

Because Node.js 14 comes with npm 6 by default, upgrade it manually
as shown in https://github.com/bahmutov/npm-install/issues/103#issuecomment-931226602.

### CODE REVIEW

**Review by adrian17 — APPROVED** [review]

**Comment by torokati44** [comment]
Could this help with the CI failures?

**Comment by torokati44** [comment]
Some other places recommend passing `--no-optional` or `--force` to `npm install` and/or `npm ci`...

File: .github/workflows/test_web.yml [hunk 1]
```diff
@@ -45,6 +45,10 @@ jobs:
         with:
           node-version: ${{ matrix.node_version }}
 
+      # Node.js 14 comes with npm 6, but we need at least npm 7.
+      - name: Upgrade npm
+        run: npm install -g npm
```
[the remark this answers is not in the document]
Comment (relrelb) [reply .github/workflows/test_web.yml:50]: We can also enforce this using https://docs.npmjs.com/cli/v8/configuring-npm/package-json#engines:
```json
{
  "engines": {
    "npm": ">=7"
  }
}
```

**Review by Herschel — APPROVED** [review]
Thank you!

File: web/package.json [hunk 2]
```diff
@@ -32,12 +37,11 @@
         "webpack-cli": "^4.9.1"
     },
     "scripts": {
-        "bootstrap": "npm install && lerna bootstrap --hoist",
-        "build": "lerna run build --stream",
+        "build": "npm run build --workspaces",
         "build:debug": "cross-env NODE_ENV=development CARGO_FEATURES=avm_debug npm run build",
```
Comment (torokati44) [inline web/package.json:41]: Shouldn't the `--workspaces` argument also have been added on this line?

Comment (relrelb) [reply web/package.json:41]: It's not needed because it uses `build` that does `npm run build --workspaces`.

Comment (torokati44) [reply web/package.json:41]: I see, thanks. It's still not entirely clear to me "which" `build` script where runs when and why. I might read up on it. Until then, I'll remove it from #5834.

**Pull request merged**